docs/en/sql-reference/functions/statistics.md

```diff
@@ -0,0 +1,18 @@
+---
+toc_priority: 69
+toc_title: Statistics
+---
+
+# Functions for Working with Statistics {#functions-for-working-with-statistics}
+
+# proportionsZTest {#proportionsztest}
+
+Applies proportion z-test to samples from two populations.
+
+**Syntax**
+
+``` sql
+proportionsZTest(successes_x, successes_y, trials_x, trials_y, confidence_level, usevar)
```

Comment:
Please add more information, what types of arguments function take, what is `usevar`.

Reply:
I wrote detailed information for the proportionsZTest().